src/map/packets/entity_update.cpp

```diff
@@ -82,7 +82,7 @@ void CEntityUpdatePacket::updateWith(CBaseEntity* PEntity, ENTITYUPDATE type, ui
             }
             if (PEntity->spawnAnimation == SPAWN_ANIMATION::SPECIAL)
             {
-                ref<uint8>(0x28) |= 0x45;
+                ref<uint8>(0x28) |= 0x04;
```

Comment:
0x04 for Quasilumins I presume?

This is an interesting thing that needs to be configurable. Currently, Dragoon Wyverns always use the same "come from the sky" animation upon spawning in range (at 50'+) even if it's not their initial spawn. The intent of this "SPAWN_ANIMATION::SPECIAL" currently was to _specifically_ override the spawn animation for dynamic entities.

I know it's out of scope of this PR, but it might be good to convert this to a mobmod so we can use any arbitrary uint16 value for spawns to accomplish animations that may change.

I could make a separate PR to add that functionality to make Dragoon's Wyverns perform properly which will also allow you to select the proper spawn animation for what you need here, too.

Reply:
The entire `0x28` bitfield needs to be figured out but I'm fairly sure that `0x04` bit is more or less "play special animation on spawn".

Reply:
oh ok so it's not that multiple animations can fill the byte (so far as we know) it's a flag that uses whatever alternate the model has baked in. that makes sense.

Reply:
I was able to figure out a few of the bits such:

- `0x01` seems to prevent entity trigger even on green name (maybe something else but this was a side-effect)
- `0x04` play special spawn (and other?) animations
- `0x40` make the entity triggerable on client

Reply:
It doesn't seem that the change here affects dynamic entities in any way. Tested with NPC module + !fafnir